lib/config/types.ts

```diff
@@ -99,7 +99,8 @@ export interface RepoGlobalConfig {
   dryRun?: boolean;
   exposeAllEnv?: boolean;
   migratePresets?: Record<string, string>;
-  privateKey?: string | Buffer;
+  privateKey?: string;
```

Comment:
I think this can be buffer, see:
https://github.com/renovatebot/renovate/blob/11694e92949171da339e54fe0d31b1f69532a6ad/lib/workers/global/config/parse/index.ts#L41-L44